lib/matplotlib/colors.py

```diff
@@ -870,13 +870,13 @@ def reversed(self, name=None):
         """
         Return a reversed instance of the Colormap.
 
-        .. note:: This function is not implemented for base class.
+        .. note:: This function is not implemented for the base class.
 
         Parameters
         ----------
         name : str, optional
-            The name for the reversed colormap. If it's None the
-            name will be the name of the parent colormap + "_r".
+            The name for the reversed colormap. If None, the
+            name is set to ```self.name``` + ```_r```.
```

Comment:
Too many backticks

```suggestion
            name is set to ``self.name + "_r"``.
```
or
```suggestion
            name is set to ``f"{self.name}_r"``.
```

Also below.